package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "metrics",
+  "version": "0.1.0",
+  "description": "Collect metrics from RPI's discord server",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "ava --verbose"
+  },
+  "ava": {
+    "nodeArguments": [
+      "--loader=esmock"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rugpullindex/metrics.git"
+  },
+  "author": "Rug Pull Index",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/rugpullindex/metrics/issues"
+  },
+  "homepage": "https://github.com/rugpullindex/metrics#readme",
+  "dependencies": {
+    "cross-fetch": "^3.1.4",
```

Comment:
I prefer to strictly pin my requirements to e.g. circumvent supply chain attacks from npm. I also [documented](https://github.com/TimDaub/commands/blob/93b1a963a94a5be100b73181f7ae7502fb9dc5d6/npm-config#L18) how you can make this a default in npm. OK?

Reply:
Okay. Do we need to change only dependencies or devDependencies too?

Reply:
IMO dependencies are good enough.

Reply:
Done the requested changes. I also realized that `.env` is now in `.gitignore`. Thus renamed it `.env.example` so we can track the changes in it.